todo_app/app.py

```diff
@@ -1,24 +1,32 @@
-from flask import Flask, render_template, request, redirect
-from todo_app.data import session_items
+from flask import Flask, render_template, request, redirect, url_for
+from todo_app.trello_board_actions import Trello_Board_Actions
 import os
+import requests
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
 @app.route('/')
 def get_items():
-    items = session_items.get_items()
-    return render_template("index.html", items=items)
+    return render_template("index.html", items=Trello_Board_Actions().getCards())
 
 @app.route('/add_item', methods=['POST'])
 def add_new_item():
     title = request.form['title']
-    session_items.add_item(title)
+    Trello_Board_Actions().addNewCard(title)
+    return redirect(os.getenv("TODO_HOSTNAME"))
+
+@app.route('/move_item/<id>', methods=['POST'])
+def move_item(id):
+    Trello_Board_Actions().updateCardStatus(id)
+    return redirect(os.getenv("TODO_HOSTNAME"))
+
+@app.route('/delete_item/<id>', methods=['POST'])
+def delete_item(id):
```

Comment:
Nothing points to this route

Reply:
@hugh-emerson , sorry couldn't understand what is needed here